lib/models/User.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:anxiety_go/models/difficulty.dart';
+
+class User {
+  final String firstName;
+  final String lastName;
+  final String email;
+  final String password;
+  final Difficulty difficultyLevel;
+
+  User({
+    required this.firstName,
+    required this.lastName,
+    required this.email,
+    required this.password,
+    required this.difficultyLevel,
+  });
+
+  // User to json
+  Map<String, Object> toJSON() {
+    return {
+      'firstName': firstName,
+      'lastName': lastName,
+      'email': email,
+      'password': password,
+      'difficultyLevel': difficultyLevel,
+    };
+  }
+
+  // User from json
+  static User fromJSON(Map<String, Object> json) {
+    return User(
+      firstName: json['firstName'] as String,
+      lastName: json['lastName'] as String,
+      email: json['email'] as String,
+      password: json['password'] as String,
+      difficultyLevel: json['difficultyLevel'] as Difficulty,
+    );
+  }
+
+  // Create a copy with
+  User copyWith({
```

Comment:
I don't think this method is required or useful.

We can add this method later if we notice that it is necessary.

Reply:
Ah yeah this method was to help if details need changed in the user, but if you are taking the approach above of removing the `final` keywords then makes sense to remove it.